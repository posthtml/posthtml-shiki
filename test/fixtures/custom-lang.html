<shiki lang="custom-diff">
-  <h1 class="text-xl">Hello</h1>
+  <h1 class="text-xl">Hello!</h1>
</shiki>
