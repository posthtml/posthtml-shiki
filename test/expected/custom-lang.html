<pre class="shiki nord" style="background-color:#2e3440ff;color:#d8dee9ff" tabindex="0"><code><span class="line"><span style="color:#ECEFF4">-</span><span style="color:#BF616A">  &#x3C;h1 class="text-xl">Hello&#x3C;/h1></span></span>
<span class="line"><span style="color:#ECEFF4">+</span><span style="color:#A3BE8C">  &#x3C;h1 class="text-xl">Hello!&#x3C;/h1></span></span></code></pre>
